ecom_test/tests/backend_tests/product_reviews/test_prodcuts_review_smoke.py

```diff
@@ -74,7 +74,9 @@ def test_verify_review_creation_fails_when_fields_duplicated(self):
 
     @pytest.mark.tcid50
     def test_review_creation_should_not_fail_if_reviewer_is_different(self):
-        """All fields have the same value but the 'reviewer' is changes. Expecting successful review creation."""
+        """All fields have the same value but the 'reviewer' is changes.
+           Expecting successful review creation.
+        """
         # Create a review for the product
         review_rs = self.product_review_helper.create_random_review_for_product(self.product_id)
 
```

Comment:
this is interesting. we called the 'crate_random_review...' function but we didn't check if response is correct.

Reply:
Yes, I need to add more assertions functions to check the response for a few more tests not only for this test only. I will rectify this issue

Reply:
I did not test using Postman, which is why I did not add response functions now I added them to verify the response.